app/Dockerfile

```diff
@@ -22,4 +22,4 @@ RUN poetry install --no-interaction --no-root --ansi
 COPY --chown=python:python . .
 RUN poetry install --no-interaction --ansi
 
-CMD ["usedgoodreads"]
+CMD ["gunicorn", "-b", "0.0.0.0:5000", "usedgoodreads.__main__:main"]
```

Comment:
Below we expose port 443 in the container to port 443 on the host. But here we listen to port 5000 in the container?